tests/integration/features/bootstrap/FeatureContext.php

```diff
@@ -1934,6 +1934,140 @@ public function userDownloadsPeersInCall(string $user, string $identifier, strin
 		Assert::assertEquals(implode("\n", $expected) . "\n", $this->response->getBody()->getContents());
 	}
 
+	#[When('/^user "([^"]*)" schedules a message to room "([^"]*)" with (\d+)(?: \((v1)\))?$/')]
+	public function userSchedulesMessageToRoom(string $user, string $identifier, int $statusCode, string $apiVersion = 'v1', ?TableNode $formData = null): void {
+		$row = $formData->getRowsHash();
+		$row['sendAt'] = (int)$row['sendAt'];
+		if (isset($row['replyTo'])) {
+			$row['replyTo'] = self::$textToMessageId[$row['replyTo']];
+		}
+		if (isset($row['threadId']) && $row['threadId'] !== '0' && $row['threadId'] !== '-1') {
+			$row['threadId'] = self::$titleToThreadId[$row['threadId']];
+		}
+
+		$this->setCurrentUser($user);
+		$this->sendRequest(
+			'POST',
+			'/apps/spreed/api/' . $apiVersion . '/chat/' . self::$identifierToToken[$identifier] . '/schedule',
+			$row
+		);
+
+		$this->assertStatusCode($this->response, $statusCode);
+		sleep(1); // make sure Postgres manages the order of the messages
+
+		$response = $this->getDataFromResponse($this->response);
+		if (isset($response['id'])) {
+			self::$textToMessageId[$row['message']] = $response['id'];
+			self::$messageIdToText[$response['id']] = $row['message'];
```

Comment:
This is dangerous as the schedule.id is not the same data as comment.id so it's possible that you overwrite data here if a single test is executed or something as message and scheduled message could have id 1?
Ah nevermind we use snowflakes already and then they should be unique